src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -6,19 +6,21 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	golog "log"
 	"os"
 	"path"
 	"strings"
 	"syscall"
 	"text/tabwriter"
 	"time"
 
+	etcd "github.com/coreos/etcd/clientv3"
 	"github.com/facebookgo/pidfile"
 	"github.com/fatih/color"
 	"github.com/gogo/protobuf/jsonpb"
 	"github.com/gogo/protobuf/types"
 	"github.com/pachyderm/pachyderm/src/client"
-	"github.com/pachyderm/pachyderm/src/client/pkg/grpcutil"
+	// "github.com/pachyderm/pachyderm/src/client/pkg/grpcutil"
```

Comment:
Remove this line for real since it's presumably not needed.

Reply:
Done